### ISSUE TITLE
Exception "Why no lastCargoEntry?" at 1/5/2024 16:43:05 -05:00

### ISSUE THREAD

**Issue by msteele999** [njthomson/SrvSurvey#29]
### What happened?

just starting E:D with the tool already running

### Version

1.0.0.0+e7a842ffaa2afeffa4d3be794dff444b015ead69

### Exception message

Why no lastCargoEntry?

### Exception stack

   at SrvSurvey.game.Game.initCargoData()
   at SrvSurvey.game.Game.initializeFromJournal(LoadGame loadEntry)
   at System.Dynamic.UpdateDelegates.UpdateAndExecuteVoid2[T0,T1](CallSite site, T0 arg0, T1 arg1)
   at SrvSurvey.game.Game.Journals_onJournalEntry(JournalEntry entry, Int32 index)

### Relevant log output

```shell
16:42:16: Game.initSystemData: playForwards 'SAASignalsFound' (1/3/2024 13:10:07)
16:42:16: Game.initSystemData: playForwards 'SAASignalsFound' (1/3/2024 13:10:07)
16:42:16: Game.initSystemData: playForwards 'SAASignalsFound' (1/4/2024 00:39:20)
16:42:16: Game.initSystemData: playForwards 'SAASignalsFound' (1/4/2024 00:39:20)
16:42:16: Game.initSystemData: playForwards 'SAASignalsFound' (1/4/2024 23:20:42)
16:42:16: Game.initSystemData: playForwards 'SAASignalsFound' (1/4/2024 23:20:42)
16:42:16: Game.initSystemData: complete 'Algol' (104389936308), bodyCount: 0
16:42:16: Game.initCargoData: rewind to last Cargo
16:42:16: Exception processing event 'LoadGame':
System.Exception: Why no lastCargoEntry?
   at SrvSurvey.game.Game.initCargoData()
   at SrvSurvey.game.Game.initializeFromJournal(LoadGame loadEntry)
   at System.Dynamic.UpdateDelegates.UpdateAndExecuteVoid2[T0,T1](CallSite site, T0 arg0, T1 arg1)
   at SrvSurvey.game.Game.Journals_onJournalEntry(JournalEntry entry, Int32 index)
16:42:31: Mode change MainMenu => Docked
```

**Comment by msteele999** [comment]
no - I started SrvSurvey first and then started E:D

**Comment by njthomson** [comment]
@msteele999 - a fix is available, please try build `1.1.0.2` - https://github.com/njthomson/SrvSurvey/releases/tag/1.1.0.2
